src/parsing/parsers/clickhouse/jison/create/create_table.test.json

```diff
@@ -107,5 +107,32 @@
     "expectedResult": {
       "lowerCase": false
     }
+  },
+  {
+    "namePrefix": "should not throw error",
+    "beforeCursor": "CREATE TABLE foo (id INT, `some` FLOAT) ENGINE = ReplacingMergeTree(update_ts); ",
```

Comment:
Case with function declaration is valid actually
[Proof](https://github.com/ClickHouse/ClickHouse/blob/1365fc91c7b7d2022509e7948dd51f2eb2dfd5cd/tests/queries/0_stateless/01906_partition_by_multiply_by_zero.sql#L9C21-L9C21)